src/app/components/Results/Results.jsx

```diff
@@ -86,33 +79,20 @@ class Results extends React.Component {
           hits !== 0 &&
           (<div className="nypl-results-sorting-controls">
              <div className="nypl-results-sorter">
-             <button aria-expanded="false" onClick={(e) => this.sortResultsBy(e)}>
-               Sort by <strong>{this.state.sortValue}</strong>
-               <svg aria-hidden="true" className="nypl-icon" preserveAspectRatio="xMidYMid meet" viewBox="0 0 68 24">
-                 <title>wedge down icon</title>
-                 <polygon points="67.938 0 34 24 0 0 10 0 34.1 16.4 58.144 0 67.938 0"></polygon>
-               </svg>
-             </button>
-             <div id="sort-menu" className="hidden">
-             <form className="sort-form">
-               <fieldset>
-                 <label htmlFor="sort-by" className="sort-legend visuallyHidden">Sort by</label>
-                    <select
-                      id="sort-by"
-                      className="sort-legend"
-                      name="sort"
-                      onChange={(e) => this.onChange(e)}
-                      value={this.state.sortValue}
-                      >
-                        <option value="relevance">relevance</option>
-                        <option value="title_asc">title (a - z)</option>
-                        <option value="title_desc">title (z - a)</option>
-                        <option value="date_asc">date (old to new)</option>
-                        <option value="date_desc">date (new to old)</option>
-                      </select>
-                    </fieldset>
-                  </form>
-                </div>
+               <button className="active" aria-expanded="false">
```

Comment:
This button needs a click handler so that the `ul`'s class to hide and show gets updated. That could be handled by adding a state to this component to keep track of that.